ReactWindows/ReactNative.Interfaces/Modules/Core/IMutableFrameEventArgs.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace ReactNative.Modules.Core
+{
+    public interface IMutableFrameEventArgs
```

Comment:
>IMutableFrameEventArgs [](start = 21, length = 22)

This interface is an implementation detail. We shouldn't move this one.

Reply:
It looks like I forgot to include the extraction of IReactChoreographer in this first commit. ReactChoreographer has several public events (IdleCallback, etc), and the interface would seemingly need to expose that type as EventHandler<IMutableFrameEventArgs>.

Reply:
Let's just make the event types for `IReactChoreographer`:
```
event EventHandler<FrameEventArgs>
```

Reply:
Or create an `IFrameEventArgs`.